rd_ui/app/scripts/visualizations/cohort.js

```diff
@@ -49,13 +56,13 @@
                             container: container,
```

Comment:
Also, because there might be existing cohort visualizations without these options, you should initialize the options with default values in the renderer.